xmodule/util/xmodule_django.py

```diff
@@ -5,9 +5,39 @@
 """
 
 
+from os import listdir
+
 import webpack_loader
 # NOTE: we are importing this method so that any module that imports us has access to get_current_request
 from crum import get_current_request
+from django.conf import settings
+from webpack_loader.loader import WebpackLoader
+
+
+class XmoduleWebpackLoader(WebpackLoader):
```

Comment:
It's standard to capitalize the "M": `XModuleWebpackLoader`

Reply:
Renamed